lib/storage/file.js

```diff
@@ -932,22 +932,29 @@ File.prototype.getMetadata = function(callback) {
  * @param {object} callback.policy - The document policy.
  *
  * @example
- * file.getSignedPolicy({
+ * var TWO_WEEKS_FROM_NOW = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
```

Comment:
The better method would be `moment.add(2, 'weeks')` which our library would handle fine. I'll add an example to the docs.